index.html

```diff
@@ -0,0 +1,289 @@
+          <img src="images/Counter.svg" alt="" aria-hidden="true">
+          <p class="bullet">Back End Dev</p>
+          <img src="images/Counter.svg" alt="" aria-hidden="true">
+          <p class="bullet">2015</p>
+      </div>
+      <p class="card-text">A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+      <ul class="skills">
+          <li>html</li>
+          <li>css</li>
+          <li>javaScript</li>
+      </ul>
+      <button class="submit" type="submit">See Project</button>
+  </div>
+</div>
+
+<div class="card work">
+  <img class= "mobile-img" src="images/avail.svg" alt="" aria-hidden="true">
+
+  <div class="content">
+      <h2>Multi-Post Stories</h2>
+      <div class="experience-section">
+          <p class="canopy">Canopy</p>
+          <img src="images/Counter.svg" alt="" aria-hidden="true">
+          <p class="bullet">Back End Dev</p>
+          <img src="images/Counter.svg" alt="" aria-hidden="true">
+          <p class="bullet">2015</p>
+      </div>
+      <p class="card-text">A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+      <ul class="skills">
+          <li>html</li>
+          <li>css</li>
+          <li>javaScript</li>
+      </ul>
+      <button class="submit" type="button">See Project</button>
+  </div>
+</div>
+
+ <!-- About Me section  -->
+    <section id="about"  class="about-me-section">
+        <div class="about">
+            <section class="about-me">
+                <h2>About <br> Myself</h2>
+                <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have 
+                    a project you need coded, don’t hestiate to contact me.</p>
+            <div class="social-platforms">
+                <h3>Lets Connect</h3>
+                <img src="images/social-media-icons.png" alt="social-media-icons" aria-hidden="true">
+              <ul>
+                  <li>
+                  <a href="https://twitter.com/RehemaMwaka1">
+                    <img
+                      src="./images/twitter.svg"
+                      class="icon"
+                      alt="twitter-icon"
+                    />
+                  </a>
+                </li>
+                 <li>
+                  <a href="https://www.linkedin.com/mwlite/in/rehema-mwaka-48a1801ab" class="icon">
+                      <img class="linkedini"
+                      src="./images/1.svg"
+                      alt="twitter-icon"
+                    />
+                    <img class="linkedin"
+                      src="./images/n.svg"
+                      alt="twitter-icon"
+                    />
+                  </a>
+                </li>
+                <li>
+                  <a href="#">
+                    <img
+                      src="./images/medium.svg"
+                      class="icon m"
```

Comment:
Its a good practice to give your attributes descriptive class names, kindly replace the `m` with a more descriptive class name or remove it if its not used